File: biblioteca/conexion.py
import mysql.connector as mysql


def conectar():
    try:
        conexion = mysql.connect(
            host='127.0.0.1',
            user='root',
            password='',
            database='libro'
        )
        print('Se ha conectado a la base de datos')
        return conexion
    except mysql.Error as err:
        print('Ha ocurrido un error', err)

File: control-gastos/node/conexion.js
const mysql = require('mysql2');

const conexion = mysql.createConnection({
  host: 'localhost',
  user: 'root',
  password: '',
  database: 'gastos',
});

conexion.connect((err, conn) => {
  if (err) {
    console.log('ha ocurrido un error en la conexion' + err);
  } else {
    console.log('se conecto correctamente a la base datos');
    return conn;
  }
});

module.exports = conexion;
